docs/walk-through/loops.md

```diff
@@ -177,6 +178,32 @@ spec:
       args: ["echo sleeping for {{inputs.parameters.seconds}} seconds; sleep {{inputs.parameters.seconds}}; echo done"]
 ```
 
+## `withSequence` example
+
+This example run a simple template multiple times using `withSequence`.
```

Comment:
```suggestion
This runs a template multiple times using `withSequence`.
```

- avoid redundant words, simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- avoid assuming a level with "simple", per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding)